### PR TITLE
[Robot] Validate affiliation settings checkboxes and drop downs can retain values upon save

### PR DESCRIPTION
**Pull request by rhakeshj**
Validate the checkboxes and drop downs can retain updated values on save

Robot WI: W-8177333, W-8177351

New/Existing Test - This is a new test

To run the test case, run this from your CCI:
cci task run robot -o suites robot/EDA/tests/browser/settings/affiliations.robot

# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: robot/EDA/tests/browser/settings/affiliations.robot [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+Test Setup      Go to EDA settings tab          Affiliations
+
+*** Test Cases ***
+Verify checkbox values can retain values on save
+    [Documentation]         Updates the checkbox values of affiliations settings tab and verifies
+    ...                     the updated values are retained upon save
+    [tags]                    unstable        W-8177333     rbt:high
+    Update checkbox value
+    ...         Enable Record Type Validation=true
```
Comment (Safiyab) [inline robot/EDA/tests/browser/settings/affiliations.robot:19]: So this keyword internally takes care of going into edit mode, update the checkbox and save ?

**Review by Safiyab — DISMISSED** [review]
Looks good ! It's great that you didn't have to add/modify any keywords to write this test. Really impressed with how you are organizing all the keywords and tests !!

File: robot/EDA/tests/browser/settings/affiliations.robot (hunk 1 above)
Comment (rhakeshj) [reply robot/EDA/tests/browser/settings/affiliations.robot:19]: Yes! It goes into edit mode and makes updates (only when necessary and saves the changes)

**Review by spinkelman — APPROVED** [review]
After a Functional Review Session with Rakesh, This item is approved.

**Pull request merged**